src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -103,16 +109,18 @@ export const useGetTradeQuotes = () => {
         error: lifiQuery.error,
         swapperName: SwapperName.LIFI,
       },
-    ].map(result => {
-      const quote = result.data && result.data.isOk() ? result.data.unwrap() : undefined
-      const inputOutputRatio = quote
-        ? getInputOutputRatioFromQuote({
-            quote,
-            swapperName: result.swapperName,
-          })
-        : -Infinity
-      return Object.assign(result, { inputOutputRatio })
-    })
+    ]
+      .filter(result => !isCrossAccountTrade || isCrossAccountTradeSupported(result.swapperName))
```

Comment:
the concept of "cross account trade" is moved into the api response rather than being tightly coupled to account ID management